charts/metallb/values.yaml

```diff
@@ -318,7 +318,7 @@ speaker:
     enabled: false
     image:
       repository: quay.io/frrouting/frr
-      tag: 7.5.1
```

Comment:
this is required because we want to keep the 7.5 format of the configuration.
The frr versoin statement declares the version of the configuration we want to apply